src/components/stateless/Logos/Logos.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './style.css'
+import shield from '../../../resources/img/work-imgs/logo.svg'
+import tugon from '../../../resources/img/work-imgs/tugon-logo.svg'
+import lions from '../../../resources/img/work-imgs/LIONS-Logo.svg'
+import uxph from '../../../resources/img/work-imgs/uxph.svg'
+
+
+const Logos = () => {
+return (
+    <div className = "mainLogos">
+                <div className = "indivLogos">
+                    <div className="indivLogo shield">       
+                     <img src={shield}></img>
+                    </div>
+                    <div className="indivLogo tugon">
+                    <img src={tugon}></img>
+
+                    </div>
+                    <div className="indivLogo lions">
+                    <img src={lions}></img>
+
+                    </div>
+                    <div className="indivLogo uxph">
+                    <img src={uxph}></img>
+
+                    </div>
```

Comment:
The images here should be props. The icons are not centered on wider resolutions: 
![image](https://user-images.githubusercontent.com/25378966/46922949-a6dfec80-cfc5-11e8-95ff-d7b19b5fdb3b.png)